helios/models.py

```diff
@@ -356,7 +354,7 @@ def sms_credentials(self):
             if not data:
                 return None
             return data.strip().split(":")
-        except:
+        except AttributeError:
```

Comment:
What is that guarding against? If it's `self.sms_data` being None, it would be better to just:
```
@property
def sms_credentials(self):
    if not self.sms_data or not self.sms_data.credentials:
        return None
    return self.sms_data.credentials.strip().split(":")
```